lib/screens/edit_product_screen.dart

```diff
@@ -141,6 +219,7 @@ class _EditProductScreenState extends State<EditProductScreen> {
                 onFieldSubmitted: (_) => focusScope.requestFocus(_descriptionFocusNode),
                 controller: _priceController,
                 validator: validatorPrice,
+                onSaved: onSavedPrice,
```

Comment:
doesn't make sense to me. you already have a controller and you still use this? why not just get the text from controller instead?

Reply:
I tried omitting the onSaved function, the edited product does not reflect.